crest/Assets/Crest/Crest/Shaders/OceanInputs/Resources/AnimWavesGerstnerBatch.shader

```diff
@@ -2,13 +2,11 @@
 
 // This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
 
-// A batch of Gerstner components
-Shader "Crest/Inputs/Animated Waves/Gerstner Batch"
+// Adds Gerstner waves everywhere. Must be given batch prepared by ShapeGerstnerBatched.cs.
+Shader "Crest/Inputs/Animated Waves/Gerstner Batch Global"
```

Comment:
Will anyone use this shader directly? It could be made hidden if not.

Reply:
No they won't - good shout

Reply:
Done